libs/community/langchain_community/chat_models/vertexai.py

```diff
@@ -289,17 +290,21 @@ def _generate(
             history_gemini = _parse_chat_history_gemini(messages, project=self.project)
             message = history_gemini.pop()
             chat = self.client.start_chat(history=history_gemini)
-            response = chat.send_message(message, generation_config=params)
+            try:
+                responses = [chat.send_message(message, generation_config=params)]
+            except ResponseBlockedError as e:
+                responses = e.responses
```

Comment:
Will break if the error case is not the successful one (GoogleCloudPlatform/generative-ai#344) without the text attribute, additionaly, we need to add some mechanism to handle unsuccessfull exception (https://github.com/langchain-ai/langchain/issues/17032)